tests/integration/test_storage_hdfs/test.py

```diff
@@ -238,11 +239,21 @@ def test_virtual_columns(started_cluster):
 def test_read_files_with_spaces(started_cluster):
     hdfs_api = started_cluster.hdfs_api
 
-    hdfs_api.write_data("/test test test 1.txt", "1\n")
-    hdfs_api.write_data("/test test test 2.txt", "2\n")
-    hdfs_api.write_data("/test test test 3.txt", "3\n")
-    node1.query("create table test (id UInt32) ENGINE = HDFS('hdfs://hdfs1:9000/test*', 'TSV')")
+    fs = HdfsClient(hosts=started_cluster.hdfs_ip)
```

Comment:
Why do we need another instance of hdfs client?

Reply:
They are different objects